pkg/util/test/util.go

```diff
@@ -440,6 +453,35 @@ func validateStorageNodes(pxImageList map[string]string, cluster *corev1.Storage
 	return nil
 }
 
+func nodeSpecsToMaps(nodes []corev1.NodeSpec) map[string]*corev1.CloudStorageNodeSpec {
+	toReturn := map[string]*corev1.CloudStorageNodeSpec{}
+
+	for _, node := range nodes {
+		toReturn[node.Selector.NodeName] = node.CloudStorage
```

Comment:
A node selector could be either nodeName or a labelSelector. If the nodeName is empty (when using labelSelector), this will not work. 
I guess it is okay for now assuming your tests are only using nodeName, but just something to remember. May be just put a TODO comment here saying this does not handle node specs with label selectors.

Reply:
Yeah, good point, let me document that.